app/auth/views/api_to_cookie.py

```diff
@@ -0,0 +1,33 @@
+import arrow
+from flask import redirect, url_for, request, flash
+from flask_login import current_user, login_user
+
+from app.auth.base import auth_bp
+from app.log import LOG
+from app.models import ApiToCookieToken
+from app.utils import sanitize_next_url
+
+
+@auth_bp.route("/api_to_cookie", methods=["GET"])
+def api_to_cookie():
+    if current_user.is_authenticated:
+        LOG.d("user is already authenticated, redirect to dashboard")
+        return redirect(url_for("dashboard.index"))
+    code = request.args.get("token")
+    if not code:
+        flash("Missing token", "error")
+        return redirect(url_for("auth.login"))
+
+    token = ApiToCookieToken.get_by(code=code)
```

Comment:
Nits: I wonder if ApiToCookieToken.code should be renamed to ApiToCookieToken.token to be more consistent?

Reply:
ApiKeys also use `code` in the db for the token. That's why I went for code here also.